src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ *   in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ *   in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
+
+  const sourceArr = sourceString
+    .replace(/\n/g, '')
+    .replace(/ +/, '')
+    .replace(/  +/g, '')
```

Comment:
not sure, maybe you can replace it with one regex expression

Reply:
Think so 2

Reply:
Try to use 1 regex expression instead of 3. Or did `trim` inside if `forEach` instead of this regex expressions.